docs/docs/api.md

```diff
@@ -0,0 +1,357 @@
+---
+title: REST API
+---
+
+# REST API
+
+You can communicate with Chronos using a RESTful [JSON][] API over HTTP.
+Chronos nodes usually listen on `port 8080` for API requests.
+All examples in this section assume that you've found a running leader at `chronos-node.airbnb.com:8080`.
+
```

Comment:
Cool, we discussed this separately. It's using Chaos and ScallopConf which gives us by default 8080. About steering clear of Mesosphere mentions, that makes sense. If you check the license file though there is ownership by Airbnb and Mesosphere.